GitUI/CommandsDialogs/FormFormatPatch.cs

```diff
@@ -2,6 +2,8 @@
 using System.IO;
 using System.Net;
 using System.Net.Mail;
+using System.Net.Security;
+using System.Security.Cryptography.X509Certificates;
```

Comment:
:bulb: Likely an unnecessary addition

Reply:
Yes. Not sure how that got there. Will remove.